examples/c/usdt.bpf.c

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+/* Copyright (c) 2022 Hengqi Chen */
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/usdt.bpf.h>
+
+SEC("usdt//lib/x86_64-linux-gnu/libc.so.6:libc:setjmp")
```

Comment:
can we use `libc.so.6` and not specify absolute path? libbpf has extra logic to try to find the library in common search path, so by relying on that this example can hopefully work on non-x86 architectures as well